src/System.Net.Http/src/System/Net/Http/Managed/HttpConnectionContent.cs

```diff
@@ -54,6 +54,11 @@ private HttpContentStream ConsumeStream()
 
             protected internal override bool TryComputeLength(out long length)
             {
+                if (Headers.ContentLength != null)
```

Comment:
ContentLength isn't a trivial property, so its result should be stored into a local, e.g.
```C#
long? contentLength = Headers.ContentLength;
if (contentLength.HasValue)
{
    length = contentLength.GetValueOrDefault();
    return true;
}

length = 0;
return false;
```